src/wordlift.php

```diff
@@ -668,6 +668,11 @@ function wl_replace_item_id_with_uri( $content ) {
 require_once( 'admin/wordlift_admin_sync.php' );
 //}
 
+/**
+ * The admin 'Install wizard' page.
+ */
+require_once ('admin/class-wordlift-install-wizard.php');
```

Comment:
Following WP's plugin boilerplate, this file must be included in `includes/class-wordlift.php`